openshift/gabi.template.yaml

```diff
@@ -71,7 +71,7 @@ objects:
           volumeMounts:
           - mountPath: /etc/tls/private
             name: gabi-tls
-        - image: quay.io/app-sre/gabi:${IMAGE_TAG}
+        - image: quay.io/redhat-services-prod/app-sre-tenant/gabi-main/gabi-main:${IMAGE_TAG}
```

Comment:
I'd first define `IMAGE` as `quay.io/app-sre/gabi` and override it in the saas file. In that way you can control better where you deploy this, the impact of this change is large.